tests/Entity/JsonPointerTest.php

```diff
@@ -92,4 +92,16 @@ public function getTestData()
 
         );
     }
+
+    public function testJsonPointerWithPropertyPaths()
+    {
+        $initial = new JsonPointer('#/definitions/date');
+        $modified = $initial->withPropertyPaths(array('~definitions/general', '%custom%'));
```

Comment:
@jojo1981 that still doesn't answer my question for more explanation about your intention. the second assert block for `$initial` already implies that it was okay in the first place - i don't see what kind of errors we could avoid by testing that (again). i'm not opposed to your suggestions, not at all, just want to understand if there is a real improvement that i'm just not seeing right now